include/hc_math.hpp

```diff
@@ -71,6 +71,8 @@ function(Q arg1) __attribute__((hc,cpu)) { \
   return hc::precise_math::function(arg1); \
 }
 
+#define HC_MATH_WRAPPER_FP_OVERLOAD_TQ_STD(function, T, arg1)  HC_MATH_WRAPPER_FP_OVERLOAD_TQ(function, T, arg1)
+
```

Comment:
Some of the math functions are only defined within the std namespace in libstdc++